### DISCUSSION TITLE
Supply SSH private key for remote builder

### DISCUSSION THREAD

**Discussion by schrockwell** [basecamp/kamal#1309] [Q&A]
When using a remote builder with an `ssh://` URL, is there any way to specify the private key? Or any other generic SSH options?

The existing options in `ssh.keys` are being ignored, and `~/.ssh/id_rsa` is being used instead.

**Comment by nickhammond** [comment]
@schrockwell You could probably put some additional configuration in your SSH agent(usually ~/.ssh/config), I don't think there's anything within Kamal at the moment though for the builder connection.

**Reply by schrockwell** [reply]
Yeah, that's an option, but it's not very "DRY" to keep config in a separate location when Kamal already supports custom SSH options for deploys. It would be nice to have it all in one spot.

**Reply by nickhammond** [reply]
Agreed, can always open a PR for it.
